src/index.html

```diff
@@ -5,8 +5,421 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--active is-active" href="#">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog page__catalog">
+        <div class="product-card catalog__product-card" data-qa="card">
+          <img
+            class="product-card__photo"
+            src="images/imac.jpeg"
+            alt="Моноблок APPLE iMac
+            27 дюймов"
+          >
+          <h3 class="product-card__name">
```

Comment:
we need to use titles in semantical rules, so if you use h3, you need to have h2 and h1 too.